workos/audit_logs.py

```diff
@@ -73,7 +73,7 @@ class AuditLogs(AuditLogsModule):
 
     _http_client: SyncHTTPClient
 
-    @validate_settings(AUDIT_LOGS_MODULE)
+    @validate_settings(Module.AUDIT_LOGS)
```

Comment:
I changed the module names to an enum a) because the typing was ending up odd for the strings, and b) because I think it's a nicer list to maintain.